From 5b61dfd69f4ecbaca6439caa9ae41b3168e4d21a Mon Sep 17 00:00:00 2001
From: Justin Jacobs <jajdorkster@gmail.com>
Date: Sat, 5 Apr 2025 18:31:03 -0400
Subject: [PATCH] Minimum CMake version = 3.5

From the CMake 4.0 release notes:
> Compatibility with versions of CMake older than 3.5 has been removed.

Upstream: https://github.com/flareteam/flare-game/commit/5b61dfd69f4ecbaca6439caa9ae41b3168e4d21a

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d98c2ae5f..9c1750552 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
 Project (Flare-Game LANGUAGES NONE)
-cmake_minimum_required (VERSION 2.8.12)
+cmake_minimum_required (VERSION 3.5)
 
 Set (PACKAGE "FLARE-GAME")
 Set (VERSION "1.14")
